client/src/components/PinnedContent/PinnedContent.js

```diff
@@ -84,24 +84,6 @@ class _PinnedContent extends React.Component {
     this.update_content_height();
   }
 
-  componentDidUpdate(prev_props, prev_state) {
-    const {
-      match: {
-        params: { options: panel_link },
-      },
-    } = this.props;
-
-    const {
-      match: {
-        params: { options: prev_panel_link },
-      },
-    } = prev_props;
-
-    !prev_panel_link &&
-      panel_link &&
-      this.setState({ user_has_enabled_pinning: false });
-  }
-
```

Comment:
Not sure if it's just me, but the pinned content now inconsistently decides to pin itself when using the ToC?

Reply:
Hmm I've been trying and never happened for me

Reply:
Not inconsistent (might appear to be if you're editing the url without reloading vs with reloading). I did drop the functionality of disabling pinning when using the table of contents though.

Was weighing taking the time to make a non-hacky (or at least less hacky) way for other components to disable pinned content, since that was a pre-existing TODO in `PinnedContent.js`. Could circle back to that in this PR and bring back that table of content feature.

Reply:
Oh I know what's causing it

If you scroll down so that the pinned content enters sight, then select a panel to go to, it will keep itself as stickied.
But if you don't have it in sight then click on a panel, it won't sticky.